packages/orama/src/types.ts

```diff
@@ -236,7 +239,7 @@ export type SearchParams<T extends AnyOrama, ResultDocument = TypedDocument<T>>
   /**
    * The properties of the document to search in.
    */
-  properties?: '*' | FlattenSchemaProperty<T>[]
+  properties?: '*' | FlattenSchemaProperties<T>
```

Comment:
Why converting this to plural form?

Reply:
Because it is an array of FlattenSchemaProperty.
Do you prefer another convention?

Reply:
Yes by convention it is better to use singular and convert them to arrays when used in lists

Reply:
done!

Reply:
done!